book/src/pong-tutorial/pong-tutorial-01.md

```diff
@@ -44,19 +43,16 @@ just implement two methods:
 # use amethyst::renderer::{DisplayConfig, DrawFlat, Pipeline,
 #                          PosTex, RenderBundle, Stage};
 # struct Pong;
-impl<'a, 'b> SimpleState<'a,'b> for Pong {
+impl<'a, 'b> SimpleState<'a, 'b> for Pong {
 }
 ```
 
-The `handle_event` method is executed for every event before updating, and it's
-used to react to events. It returns a `Trans`, which is an enum of state machine
-transitions. In this case, we're watching for the Escape keycode, and the
-`CloseRequested` event from the Window. If we receive it, we'll return
-`Trans::Quit` which will be used to clean up the `State` and close the application.
-All other keyboard input is ignored for now.
-
-The `update` method is executed after events have happened.  In this instance
-we're just using it to execute our dispatcher.  More on that later.
+The `SimpleState` already implements a bunch of stuff for us, like the `update` 
+and `handle_event` methods that you would have to implement yourself were you 
+using just a simple `State`. In particular, the default implementation for
```

Comment:
change `simple State` to `normal State` or something. I feel it might be misunderstood as `simple State` is fairly close to `SimpleState`